### DISCUSSION TITLE
The tensorflow version question

### DISCUSSION THREAD

**Discussion by hongyingyue** [PaddlePaddle/PaddleSpatial#51] [BaiduKDDCup2022] [answered]
As [tensorflow](https://www.tensorflow.org/install/gpu) mentioned, the current version of tensorflow-gpu==2.1.0 and the mentioned CUDA version 11.2 is not matched. So I guessed the tensorflow-gpu need at least higher than 2.5 to use GPU, thanks

**Comment by ramber1836** [comment, marked as answer]
We are working on it. 
Please notice the updates to the Evaluation Manual.
